credsweeper/common/morpheme_checklist.txt

```diff
@@ -93,6 +93,7 @@ adapt
 add
 ader
 adjust
+admi
```

Comment:
I think it can make filter the line which contain `admin` keyword.. like `admin_key`, `admin_secret`, and so on..

Reply:
Yes. The list predestination is to filter out true random tokens and human typed example tokens. In true random token the sequence 'admin'  is too rarely.

Reply:
Ah.. I mistook it for checking `variable`..
It just check the `value` and yes, `admin` should to included in this checklist.